tracing-subscriber/src/fmt/time/chrono_crate.rs

```diff
@@ -0,0 +1,57 @@
+use crate::fmt::format::Writer;
+use crate::fmt::time::FormatTime;
+
+/// Formats [local time]s and [UTC time]s with [formatter] implementations
+/// that use the [`chrono` crate].
+///
+/// [local time]: https://docs.rs/chrono/0.4.26/chrono/offset/struct.Local.html
+/// [UTC time]: https://docs.rs/chrono/0.4.26/chrono/offset/struct.Utc.html
+/// [`chrono` crate]: https://docs.rs/chrono/0.4.26/chrono/
+/// [formatter]: https://docs.rs/time/0.3/time/formatting/trait.Formattable.html
+
+/// Tag-type (indicating UTC timezone) enabling static dispatch
+/// to `chrono::Local` functions.
+#[derive(Debug)]
+pub struct LocalTime;
```

Comment:
You'll need to rename other usage sizes as well:

```suggestion
/// Retrieve and print the current local time.
#[cfg(feature = "chrono")]
#[cfg_attr(docsrs, doc(cfg(feature = "chrono")))]
#[derive(Debug, Clone, Eq, PartialEq, Default)]
pub struct ChronoLocal;
```